cuda/CMakeLists.txt

```diff
@@ -129,6 +129,33 @@ target_sources(ginkgo_cuda
 foreach(source_file IN LISTS GKO_CUDA_COMMON_SOURCES)
     set_source_files_properties(${source_file} PROPERTIES LANGUAGE CUDA)
 endforeach(source_file)
+if(GINKGO_JACOBI_FULL_OPTIMIZATIONS)
+    set(GKO_CUDA_JACOBI_BLOCK_SIZES)
+    foreach(blocksize RANGE 1 32)
+        list(APPEND GKO_JACOBI_BLOCK_SIZES ${blocksize})
```

Comment:
```suggestion
        list(APPEND GKO_CUDA_JACOBI_BLOCK_SIZES ${blocksize})
```

Reply:
but why is it not caught by the test?

Reply:
Oh, good catch! Probably because we don't build with full optimizations in any tests?